observatory-platform/observatory/platform/workflows/templates/workflow_stream.py.jinja2

```diff
@@ -47,13 +47,14 @@ class {{ workflow_class }}(StreamTelescope):
         dataset_id: str = "your_dataset_id",
         dataset_description: str = "The your_dataset_name dataset: https://dataseturl",
         merge_partition_field: str = "id",
+        workflow_id: int = 0,
```

Comment:
Could we make this a positional argument rather than a keyword argument? Since you have to specify it every time you instantiate a StreamTelescope subclass.

Reply:
Yeah sure, you mean just in this jinja2 template? It is already a positional argument in the actual StreamTelescope class. This jinja2 template is for an example workflow that uses the StreamTelescope. For the actual existing telescopes that use the StreamTelescope the workflow_id is set in the DAG file, so this template is an exception I guess.

Reply:
Yeah in the jinja2 template and the other StreamTelescope subclasses, but I'm not sure if it will cause a lot of changes to the tests for the other StreamTelescope subclasses?

Reply:
I will have a look :-)